### PR TITLE
Bump django-jsonview

### PR DESCRIPTION
**Pull request by JBKahn**
Improved Django 1.8 compatibility.
It didn't drop any compatibility either
https://github.com/jsocol/django-jsonview/blob/ae393b13c54ee3a3eb1a1437ea44ea0adfbdf059/CHANGELOG#L11

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ env:
   - DJANGO_VERSION=1.4.15
   - DJANGO_VERSION=1.6.7
   - DJANGO_VERSION=1.7.0
+  - DJANGO_VERSION=1.8.3
```
Comment (mwarkentin) [inline .travis.yml]: Would you mind actually matching this up with the officially supported Django versions? Currently 1.4.x, 1.7.x, and 1.8.x - feel free to bump the 1.4.x / 1.7.x to the latest release as well.

Comment (JBKahn) [inline .travis.yml]: Will do.

**Comment by mwarkentin** [comment]
Just the one comment, otherwise LGTM.

:+1:

**Comment by mwarkentin** [comment]
@JBKahn Looks good - is this something you need a release of quickly?

**Comment by JBKahn** [comment]
@mwarkentin when you have time it would be good :) but I suppose it can wait.

**Pull request merged**